py-polars/polars/lazyframe/frame.py

```diff
@@ -1872,7 +1877,7 @@ def collect(
         collapse_joins: bool = True,
         no_optimization: bool = False,
         streaming: bool = False,
-        engine: EngineType = "cpu",
+        engine: EngineType = "auto",
```

Comment:
We should deprecate the `streaming` flag on `collect`. 

`streaming = True` will still dispatch to old streaming engine for backwards compatibility.

`engine = "streaming"` will dispatch to new streaming engine.